docs/get-started/network-information.mdx

```diff
@@ -6,10 +6,96 @@ import InlineCopy from '@site/src/components/InlineCopy';
 
 For current and historical status information for Etherlink, see https://status.etherlink.com.
 
+For every network, we provide two endpoints: a public RPC endpoint and a relay
+endpoint. The public RPC endpoint can be consumed by Etherlink users and
+decentralized applications to interact with the chain. The relay endpoint
+exposes a restricted RPC API with only `eth_sendRawTransaction` and
+`eth_blockNumber` available. The relay endpoint should be preferred when
+setting up an [Etherlink node](/network/evm-nodes).
+
+:::warning Public RPC Endpoint Restrictions
```

Comment:
Should we also mention these limitations (batch limit and supported endpoints) on the "Ethereum endpoint support" page?

Reply:
I’m not sure, it’s not directly related to the software we distribute and deploying your own node allow to not have them.

But no strong opinion either.